### DISCUSSION TITLE
how

### DISCUSSION THREAD

**Discussion by Asal73** [meetric1/gmod-infinite-map#79] [Q&A]
how do i download it?

**Comment by meetric1** [comment]
Download code from the main page and extract it to your gmod addons folder

**Reply by Asal73** [reply]
what is the map called after i downloaded it? i cant find it in my maps

**Reply by TheVogels** [reply]
> what is the map called after i downloaded it? i cant find it in my maps
Gm_infmap is main, Gm_infmap_empty is empty infinite map.
